servicetalk-examples/grpc/helloworld/src/main/resources/log4j2.xml

```diff
@@ -21,14 +21,6 @@
     </Console>
   </Appenders>
   <Loggers>
-    <!--
-        This is an example of how to configure logging of wire events:
-          1. Add a Logger with `TRACE` level;
```

Comment:
since there is no more strict requirement on the log level, we have an opportunity to simplify the examples. Log4j can be configured in many ways (filters, multiple appenders, using different loggers, etc.) and I don't think this is central to understanding servicetalk.